test/e2e_node/util.go

```diff
@@ -408,6 +408,13 @@ func restartKubelet(running bool) {
 
 	stdout, err = exec.Command("sudo", "systemctl", "restart", kubeletServiceName).CombinedOutput()
 	framework.ExpectNoError(err, "Failed to restart kubelet with systemctl: %v, %s", err, string(stdout))
+
+	// wait until the kubelet status is stable
```

Comment:
I think you need the `Eventually` first, so
- wait until kubelet is ready
- once is ready, check it is always ready

Reply:
Misunderstand that. 

Not sure. `If it can run consistently for 10s, I thought it is enough.`

Reply:
I've made up the numbers, don't know what values are the best here honestly